src/store/api/people.js

```diff
@@ -274,18 +274,21 @@ export default {
     const data = {
       name: searchFilter.name,
       search_query: searchFilter.search_query,
-      search_filter_group_id: searchFilter.search_filter_group_id
+      search_filter_group_id: searchFilter.search_filter_group_id,
+      is_shared: searchFilter.is_shared,
+      project_id: searchFilter.project_id
     }
     return client.pput(`/api/data/user/filters/${searchFilter.id}`, data)
   },
 
-  createFilter(listType, name, query, productionId, entityType) {
+  createFilter(listType, name, query, productionId, entityType, isShared) {
```

Comment:
In the pull-request, it is missing the `src/store/modules/*.js` files that call the method with this new parameter, right?